src/Microsoft.OpenApi/Models/OpenApiDocument.cs

```diff
@@ -544,13 +544,17 @@ private static string ConvertByteArrayToString(byte[] hash)
         {
             // Build the final string by converting each byte
             // into hex and appending it to a StringBuilder
+#if NET
```

Comment:
```suggestion
#if NET5_0_OR_GREATER
```
Since this is only available starting in net5. https://learn.microsoft.com/en-us/dotnet/api/system.convert.tohexstring?view=net-9.0#system-convert-tohexstring(system-byte())
( in case we ever decide to change the targets for whatever reason)

Reply:
My understanding is that `NET` means exactly the same thing - .NET Core 3.1 and lower are `NETCORE` (not `NET`) and there's no such thing as .NET (Core) 4, so the two directives are identical, except for their length in characters.